types/Actions.d.ts

```diff
@@ -204,185 +350,185 @@ declare module 'grafx-studio-actions' {
          * @param height the frame height
          * @returns
          */
-        resize(name: string, width: number, height: number): void;
+        resize(name: string | Frame, width: number | VariableValue, height: number | VariableValue): void;
 
         /**
          * Moves a frame to a specified X position
          * @param name the frame name to move
          * @param x the frame x position
          * @returns
          */
-        setX(name: string, x: number): void;
+        setX(name: string | Frame, x: number | VariableValue): void;
 
         /**
          * Moves a frame to a specified Y position
          * @param name the frame name to move
          * @param y the frame y position
          * @returns
          */
-        setY(name: string, y: number): void;
+        setY(name: string | Frame, y: number | VariableValue): void;
 
         /**
          * Resizes a frame to a specified width
          * @param name the frame name to resize
          * @param width the frame width
          * @returns
          */
-        setWidth(name: string, width: number): void;
+        setWidth(name: string | Frame, width: number | VariableValue): void;
 
         /**
          * Resizes a frame to a specified height
          * @param name the frame name to resize
          * @param height the frame height
          * @returns
          */
-        setHeight(name: string, height: number): void;
+        setHeight(name: string | Frame, height: number | VariableValue): void;
 
         /**
          * Rotates a frame to a specified degree
          * @param name the frame name to rotate
          * @param rotation the frame rotation in degrees
          * @returns
          */
-        setRotation(name: string, rotation: number): void;
+        setRotation(name: string | Frame, rotation: number | VariableValue): void;
 
         /**
          * Toggles the frame visibility
          * @param name the frame name to include/exclude
          * @param include whether the frame is included or not
          * @returns
          */
-        include(name: string, include: boolean): void;
+        include(name: string | Frame, include: boolean | VariableValue): void;
```

Comment:
@Matthiee verify this is correct after https://github.com/chili-publish/studio-sdk/pull/327 is merged.